### PR TITLE
Feature/merge base config

### PR DESCRIPTION
**Pull request by yurii-klymenko**
configurations:
 - remove module name for lang.json and conf.json
 - add type:fields
 - add displayName
ability to package one module only
flake8 formatting
merge basic conf/lang during packing

### CODE REVIEW

**Comment by delliott90** [comment]
Do you need to add `jsonmerge` to the requirements? The setup script doesn't work without it.

File: stix_shifter_modules/azure_sentinel/configuration/config.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
     "connection": {
         "type": {
-            "default": "azure_sentinel"
+            "default": "azure_sentinel",
+            "displayName": "Amazon CloudWatch Logs"
```
Comment (mdazam1942) [inline stix_shifter_modules/azure_sentinel/configuration/config.json:5]: "displayName": "Microsoft Azure Sentinel"

File: stix_shifter_modules/bigfix/configuration/config.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
     "connection": {
         "type": {
-            "default": "BigFix"
+            "default": "bigfix",
+            "displayName": "IBM® BigFix"
```
Comment (mdazam1942) [inline stix_shifter_modules/bigfix/configuration/config.json:5]: `"displayName": "BigFix"`

its not an IBM product anymore. so we need to remove all mentions of IBM with BigFIx

File: stix_shifter_modules/carbonblack/configuration/config.json [hunk 3]
```diff
@@ -1,7 +1,8 @@
 {
     "connection": {
         "type": {
-            "default": "CarbonBlack"
+            "default": "carbonblack",
+            "displayName": "Carbon Black CB Response"
```
Comment (mdazam1942) [inline stix_shifter_modules/carbonblack/configuration/config.json:5]: "displayName": "Carbon Black Response"

I know `CB` is there in the existing display name but I don't understand why its there as we already mentioned `Carbon Black`

**Pull request merged**